### PR TITLE
MSIZE

### PR DESCRIPTION
**Pull request by npwardberkeley**

### CODE REVIEW

File: evm/src/cpu/kernel/interpreter.rs [hunk 1]
```diff
@@ -511,6 +511,14 @@ impl<'a> Interpreter<'a> {
         }
     }
 
+    fn run_msize(&mut self) {
+        let num_u256s = self.memory.context_memory[0].segments[Segment::MainMemory as usize]
+            .content
+            .len();
+        let num_bytes = num_u256s * 32;
```
Comment (dlubarov) [inline evm/src/cpu/kernel/interpreter.rs:518]: In our model we aren't packing several memory bytes into one U256, but rather just using one byte of the U256, and leaving the rest as zero. So we don't need `* 32`

File: evm/src/cpu/kernel/interpreter.rs [hunk 2]
```diff
@@ -511,6 +511,14 @@ impl<'a> Interpreter<'a> {
         }
     }
 
+    fn run_msize(&mut self) {
+        let num_u256s = self.memory.context_memory[0].segments[Segment::MainMemory as usize]
```
Comment (dlubarov) [inline evm/src/cpu/kernel/interpreter.rs:515]: Let's use `self.context` instead of `0` to make it more general

**Review by dlubarov — APPROVED** [review]
LGTM otherwise

File: evm/src/cpu/kernel/interpreter.rs (hunk 1 above)
Comment (npwardberkeley) [reply evm/src/cpu/kernel/interpreter.rs:518]: Ohhhh right of coruse

**Pull request merged**